src/libsystemd/sd-netlink/netlink-message.c

```diff
@@ -343,6 +343,74 @@ int sd_netlink_message_append_u64(sd_netlink_message *m, unsigned short type, ui
         return 0;
 }
 
+int sd_netlink_message_append_s8(sd_netlink_message *m, unsigned short type, int8_t data) {
```

Comment:
Already included in TC . https://github.com/systemd/systemd/pull/14890/commits/1330553dc62d30e232935a4c146fdb5d3e3a43f1#diff-a7c9d8d44082fe9a024afe9ad52db884R346

Reply:
Yes, I will rebase this once the branch is merged.

Reply:
I like this work. Thanks. We should have done this earlier.